misc/acrn-config/xmls/board-xmls/apl-up2.xml

```diff
@@ -264,18 +264,18 @@
 	00001000-0003efff : System RAM
 	00040000-0009dfff : System RAM
 	00100000-0fffffff : System RAM
-	12151000-7074c017 : System RAM
-	7074c018-7075c057 : System RAM
-	7075c058-77b30fff : System RAM
-	7a09d000-7a40afff : System RAM
-	7a426000-7a964fff : System RAM
+	12151000-70884017 : System RAM
+	70884018-70894057 : System RAM
+	70894058-77b0afff : System RAM
+	7a08b000-7a3f8fff : System RAM
+	7a424000-7a964fff : System RAM
 	7a967000-7affffff : System RAM
-	100000000-17fffffff : System RAM
+	100000000-27fffffff : System RAM
 	</SYSTEM_RAM_INFO>
 
 	<BLOCK_DEVICE_INFO>
-	/dev/mmcblk0p3: UUID="bb0d14f3-e780-41d9-bcca-6f3ef493216c" TYPE="ext4" PARTLABEL="primary" PARTUUID="87cd7180-6c0b-4bc9-a0d0-5406a95988cc"
-	/dev/sda3: TYPE="ext4"
+	/dev/sda3: LABEL="root" UUID="7de2c654-069b-4a56-b93f-70b2f7662bd0" TYPE="ext4" PARTLABEL="/" PARTUUID="712932d4-3420-4cca-b361-3606a40f3b89"
```

Comment:
for UUID, each board or partition will have a different UUID, right? how is it work for different boards?

Reply:
Actually we did not care about UUID here. The whole line is copy directly from "blkid" tool.

Reply:
Do you mean uuid is not used?